css-typed-om/Overview.bs

```diff
@@ -2897,6 +2897,13 @@ depending on the property they came from:
     8. Return the result of concatenating all the items in |values|,
         separated by " " (U+0020 SPACE).
 
+: 'box-sizing'
+::
+    1. Return the string representation of the keyword:
```

Comment:
Just serialize the value as a keyword. No need to specify the keywords; it just means we'd have to edit this when new keywords are added.

Reply:
Done.